deepcell/datasets/__init__.py

```diff
@@ -42,128 +45,80 @@ class Dataset:
             (relative to ~/.keras/datasets).
         url (str): URL of dataset in S3.
         file_hash (str): md5hash for checking validity of cached file.
-        metadata (dict): miscellaneous other data for dataset
+        secure (bool): True if the dataset requires a deepcell api key for download. Default False
     """
 
     def __init__(self,
-                 path,
+                 basename,
                  url,
                  file_hash,
-                 metadata):
-        self.path = path
+                 secure=False):
+        self.data_dir = os.path.expanduser(os.path.join('~', '.keras', 'datasets'))
```

Comment:
We might want to consider changing this to `~/.deepcell/datasets` with a corresponding `~/.deepcell/models`